File: mvvmframe/src/main/java/com/king/frame/mvvmframe/config/FrameConfigModule.java
package com.king.frame.mvvmframe.config;


/**
 * @author <a href="mailto:jenly1314@gmail.com">Jenly</a>
 */
public abstract class FrameConfigModule implements AppliesOptions {

    /**
     * 是否启用解析配置
     * @return 默认返回{@code true}
     */
    public boolean isManifestParsingEnabled() {
        return true;
    }

}

File: mvvmframe/src/main/kotlin/com/king/frame/mvvmframe/config/FrameConfigModule.kt
package com.king.frame.mvvmframe.config

/**
 * MVVMFrame全局配置
 *
 * @author <a href="mailto:jenly1314@gmail.com">Jenly</a>
 * <p>
 * <a href="https://github.com/jenly1314">Follow me</a>
 */
abstract class FrameConfigModule : AppliesOptions {
    /**
     * 是否启用解析配置
     * @return 默认返回{@code true}
     */
    open fun isManifestParsingEnabled(): Boolean {
        return true
    }
}